recipes/cdat_info/build.sh

```diff
@@ -0,0 +1,214 @@
+export CFLAGS="-Wall -g -m64 -pipe -O2  -fPIC"
```

Comment:
Can you try `export CFLAGS="-Wall -g -m64 -pipe -O2  -fPIC $CFLAGS"` so we don't loose the pre-defined `CFLAGS`? Same is valid for all the env vars below.